sdk/subscription/Azure.ResourceManager.Subscription/api/Azure.ResourceManager.Subscription.netstandard2.0.cs

```diff
@@ -166,177 +177,106 @@ public partial class EnabledSubscriptionId
     }
-    public partial class GetTenantPolicyResponseCreateOrUpdateContent
-    {
-        public GetTenantPolicyResponseCreateOrUpdateContent() { }
-        public bool? BlockSubscriptionsIntoTenant { get { throw null; } set { } }
-        public bool? BlockSubscriptionsLeavingTenant { get { throw null; } set { } }
-        public System.Collections.Generic.IList<System.Guid> ExemptedPrincipals { get { throw null; } }
-    }
-    public partial class Location
+    public partial class RenamedSubscriptionId
     {
-        internal Location() { }
-        public string DisplayName { get { throw null; } }
-        public string Id { get { throw null; } }
-        public string Latitude { get { throw null; } }
-        public string Longitude { get { throw null; } }
-        public string Name { get { throw null; } }
+        internal RenamedSubscriptionId() { }
         public string SubscriptionId { get { throw null; } }
     }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct Provisioning : System.IEquatable<Azure.ResourceManager.Subscription.Models.Provisioning>
-    {
-        private readonly object _dummy;
-        private readonly int _dummyPrimitive;
-        public Provisioning(string value) { throw null; }
-        public static Azure.ResourceManager.Subscription.Models.Provisioning Accepted { get { throw null; } }
-        public static Azure.ResourceManager.Subscription.Models.Provisioning Pending { get { throw null; } }
-        public static Azure.ResourceManager.Subscription.Models.Provisioning Succeeded { get { throw null; } }
-        public bool Equals(Azure.ResourceManager.Subscription.Models.Provisioning other) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override bool Equals(object obj) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.ResourceManager.Subscription.Models.Provisioning left, Azure.ResourceManager.Subscription.Models.Provisioning right) { throw null; }
-        public static implicit operator Azure.ResourceManager.Subscription.Models.Provisioning (string value) { throw null; }
-        public static bool operator !=(Azure.ResourceManager.Subscription.Models.Provisioning left, Azure.ResourceManager.Subscription.Models.Provisioning right) { throw null; }
-        public override string ToString() { throw null; }
-    }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct ProvisioningState : System.IEquatable<Azure.ResourceManager.Subscription.Models.ProvisioningState>
+    public partial class ServiceTenantResponse
```

Comment:
It's better to change the suffix `Response`.

Reply:
drop the suffix